app/src/main/res/values/strings.xml

```diff
@@ -92,6 +93,7 @@
         <b>Reddit:</b>
         https://www.reddit.com/r/SimpleMobileTools
     </string>
+    <string name="password_leave_blank_if_not_needed">Password (leave blank if not needed)</string>
```

Comment:
if you add new strings, add them into all languages at the proper section please, not at the end